src/server/controller/cycleData/updateOriginalDataPoint/updateDependantsProps.ts

```diff
@@ -0,0 +1,15 @@
+import { CountryIso } from '@meta/area'
+import { Assessment, Cycle, OriginalDataPoint } from '@meta/assessment'
+import { User } from '@meta/user'
+
+import { BaseProtocol } from '@server/db'
+
+export type updateDependantsProps = {
+  countryIso: CountryIso
+  assessment: Assessment
+  cycle: Cycle
+  user: User
+  colName: string
+  updatedOriginalDataPoint: OriginalDataPoint
```

Comment:
can't this be called originalDataPoint directly ? also, use upper case for types